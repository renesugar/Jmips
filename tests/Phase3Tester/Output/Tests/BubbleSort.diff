--- SelfTestCases/BubbleSort.out	2019-02-02 14:36:13.000000000 -0800
+++ ./Output/Tests/BubbleSort.out	2019-03-11 00:03:07.000000000 -0700
@@ -1,22 +1,5 @@
-20
-7
-12
-18
-2
-11
-6
-9
-19
-5
-99999
-2
-5
-6
-7
-9
-11
-12
-18
-19
-20
-0
+--- Error (ProgramFault) ---
+require data address, found integer44
+  Main() line 17
+  BBS.Start() line 29
+  BBS.Init() line 185
