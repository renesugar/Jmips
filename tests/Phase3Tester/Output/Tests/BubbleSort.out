--- Error (ProgramFault) ---
require data address, found integer44
  Main() line 17
  BBS.Start() line 29
  BBS.Init() line 185
